src/shared/mount-util.c

```diff
@@ -1231,8 +1230,6 @@ static int mount_in_namespace(
                 }
                 if (r < 0) {
                         (void) write(errno_pipe_fd[1], &r, sizeof(r));
-                        errno_pipe_fd[1] = safe_close(errno_pipe_fd[1]);
```

Comment:
Why is this being removed?

Reply:
We're `_exit()`ing anyway, so this is unnecessary. We don't do this in other places either.

Reply:
I have vague memories that when calling exit the cleanup attrs were not ran, so if there's no explicit reason to remove it, please leave it

Reply:
Hmm? This is not about cleanup, but all fds are automatically closed when the process exits?

Reply:
I don't remember the details, but I do remember wasting time due to some issue along these lines, so please leave it as-is, at worst it doesn't make any difference

Reply:
See `dissected_image_acquire_metadata` as an example.

Reply:
Still, I don't think this would cause any trouble. It would have surfaced at other places if it was the case.

Reply:
I am pretty sure it did, and that it took a large amount of time to solve, so again given it doesn't make any difference otherwise, please leave it